scripts/pylib/twister/twisterlib/runner.py

```diff
@@ -143,49 +145,60 @@ def _find_number_length(n):
         return length
 
     def summary(self):
-        executed_cases = self.cases - self.skipped_cases - self.filtered_cases
-        completed_configs = self.done - self.skipped_filter
+        selected_cases = self.cases - self.filtered_cases
+        completed_configs = self.done - self.filtered_static - self.filtered_runtime
 
         # Find alignment length for aesthetic printing
         suites_n_length = self._find_number_length(self.total if self.total > self.done else self.done)
-        processed_suites_n_length = self._find_number_length(self.done)
         completed_suites_n_length = self._find_number_length(completed_configs)
-        skipped_suites_n_length = self._find_number_length(self.skipped_configs)
+        filtered_suites_n_length = self._find_number_length(self.filtered_configs)
         total_cases_n_length = self._find_number_length(self.cases)
-        executed_cases_n_length = self._find_number_length(executed_cases)
+        selected_cases_n_length = self._find_number_length(selected_cases)
 
         print("--------------------------------------------------")
         print(f"{'Total test suites: ':<23}{self.total:>{suites_n_length}}") # actually test instances
         print(f"{'Processed test suites: ':<23}{self.done:>{suites_n_length}}")
-        print(f"├─ {'Filtered test suites (static): ':<37}{self.skipped_filter:>{processed_suites_n_length}}")
-        print(f"└─ {'Completed test suites: ':<37}{completed_configs:>{processed_suites_n_length}}")
-        print(f"   ├─ {'Filtered test suites (at runtime): ':<37}{self.skipped_runtime:>{completed_suites_n_length}}")
+        print(f"└─{'Filtered test suites: ':<21}{self.filtered_configs}")
```

Comment:
Add space after `└─`. Change `└` into `├`. Change `21` into a number bigger than the previous tree level (`21`), e.g. `23`. Right-align the numbers so that the same tree level numbers can be easily found.
`print(f"├─ {'Filtered test suites: ':<23}{self.filtered_configs:>{suites_n_length}}")`